cluster.go

```diff
@@ -455,16 +506,58 @@ func (c *clusterClient) _pickMulti(multi []Completed) (retries *connretry, last
 	defer c.mu.RUnlock()
 
 	count := conncountp.Get(len(c.conns), len(c.conns))
-	for _, cmd := range multi {
-		if cmd.Slot() == cmds.InitSlot {
-			init = true
-			continue
+	var connIndexes []conn
```

Comment:
I would like to avoid this `connIndexes` allocation. I think we can assume the `SendToReplicas` is cheap for calling multiple times.